components/PortraitPlayingCard/PortraitPlayingCard.tsx

```diff
@@ -52,13 +52,15 @@ const Spanner = styled.div`
 const _RankIcon = styled(RankIcon)`
   position: absolute;
   top: 10%;
-  left: 22.5%;
+  left: 50%;
   width: 55%;
+  transform: translateX(-50%);
 `;
 
 const _SuitIcon = styled(SuitIcon)`
   position: absolute;
   bottom: 8%;
-  left: 20%;
+  left: 50%;
   width: 60%;
+  transform: translateX(-50%);
```

Comment:
What is this?🤔

Reply:
Hmmm how can I put it...? Move the element to the left for 50% of its width 🤔